replication-api-impl/src/test/java/com/connexta/replication/api/impl/persistence/pojo/FilterIndexPojoTest.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.api.impl.persistence.pojo;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.jparams.verifier.tostring.NameStyle;
+import com.jparams.verifier.tostring.ToStringVerifier;
+import java.time.Instant;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FilterIndexPojoTest {
+
+  private static final Instant MODIFIED_SINCE = Instant.now();
+
+  private static final String ID = "id";
+
+  private static final String FILTER_ID = "filterId";
+
+  private static final FilterIndexPojo POJO =
+      new FilterIndexPojo().setId(ID).setModifiedSince(MODIFIED_SINCE).setFilterId(FILTER_ID);
+
+  private FilterIndexPojo pojo;
+
+  @Before
+  public void setup() {
+    pojo = new FilterIndexPojo();
```

Comment:
✏️ Doesn't need a setup section for that; simply do the new above where it is defined